Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM cimg/python:3.9
+ARG VERSION=3.9.1
+ARG NAME=records-mover
+ENV version_env=$VERSION
+ENV name_env=$NAME
+WORKDIR /records-mover
+RUN git clone https://github.com/bluelabsio/records-mover.git .
```

Comment:
@naswierczek I wonder if it would be better using this:
```dockerfile
COPY . $HOME/records-mover/
```

to use the current directory instead of the existing records-mover dir to build

Reply:
okay, here's a proposed alternative to the cloning from git strategy, this is the whole Dockerfile:

```dockerfile
FROM cimg/python:3.9
ARG VERSION=3.9.1
ARG NAME=records-mover
ENV version_env=$VERSION
ENV name_env=$NAME

USER root

COPY . $HOME/records-mover/
WORKDIR /records-mover

RUN python -m venv venv \
    && . venv/bin/activate \
    && pip install --upgrade pip \
    && pip install --progress-bar=off -r requirements.txt \
    && pip install --upgrade --progress-bar=off 'pandas>=1.5.3,<2' \
    && pip install -e '.[unittest,typecheck]' #
```

Reply:
@ryantimjohn I considered this at length. Given that this is intended to support local dev, it probably does make sense to assume this dockerfile would be built in a directory where the relevant code was already checked out.

It would make sense to leave it as is if this was being used as part of a build pipeline.
Worth it to change it?

Reply:
one possibility if we want it to be specifically for local dev is renaming the file to `Dockerfile.dev` then add to the docker build statement `docker build -f Dockerfile.dev`. I've seen that in a number of repos where you want specifically a Dockerfile for development and not deployment.

The reason that I brought up building the image from local files rather than what was cloned from the main branch was that I was working on a PR where I had to update the `setup.py` with new build packages, and realized that was very hard with the clone from git hard-coded. Another option might be to have an arg for building it from a particular branch, but having that branch default to main.

Anyway, just some thoughts. This is acceptable as exists aside from the quality issues and maybe adding a little to the readme.

If we do allow building from non-main branches or from the local dir, we should def remove the tests as non-main branches and local files are often not going to pass tests as we're building on them which is the point. =)

Reply:
RE: Quality issues.
I've fixed all the ones in the README section I added but there are other parts of the README and other MD files getting flagged.

Do you think I fix those as well? They don't seem that severe.